OSDOCS-9826: adds Y+2 update paths MicroShift (#73988)
openshift/openshift-docs: microshift_updating/microshift-update-rpms-manually.adoc

[IMPORTANT]
====

> {microshift-short} updates are supported from a maximum of two minor versions. For example, you can update 4.14 to 4.16 in a single step. Updating 4.14 to 4.15 first is not required.

Reviewer: I think the earlier note reads better. 
```suggestion
Updates of {microshift-short} are supported with a maximum of two minor versions. For example, you can update from 4.14 to 4.16 in a single step without updating to 4.15 first .
```
